lib/read_file.js

```diff
@@ -6,7 +6,7 @@ function promisfy (mod, fn) {
     return new Promise(function (resolve, reject) {
       args.push(function (err, res) {
         if (err) reject(err)
-        else resolve(res)
+        else resolve(res.toString())
```

Comment:
alternatively (and I think cleaner) you can specify 'utf8' as the encoding to 'readFIle' and it will set it as a string automatically